printer.go

```diff
@@ -145,9 +146,12 @@ func Format(v interface{}) string {
 }
 
 var newLine = []byte{'\n'}
+var mux = &sync.Mutex{}
```

Comment:
This can just be a simple variable declaration, no need for the pointer.

```suggestion
var mux sync.Mutex
```